lib/routes/retell.js

```diff
@@ -15,7 +15,7 @@ const service = ({logger, makeService}) => {
     logger.info({session}, `new incoming call: ${session.call_sid}`);
 
     /* Send ping to keep alive websocket as some platforms timeout, 25sec as 30sec timeout is not uncommon */
-    session.keepAlive = setInterval(() => {
+    session.locals.keepAlive = setInterval(() => {
       session.send('ping');
```

Comment:
actually, I am not even sure this works.  The session.send() method is not expecting a string.  This is probably not the way we want to do a websocket ping.  The websocket protocol includes a 'ping' frame, which is probably what we shoould be doing.  That should be able to be simply done as

```
session.ws.ping();
```

could you test this?